script.js

```diff
@@ -1,4 +1,47 @@
+//Create a function to generate random words from a list
+var getWords = function () {
+  var listWords = ["scissors", "paper", "stone"];
+  var randomWords = listWords[Math.floor(Math.random() * listWords.length)];
+  return randomWords;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var randomSPS = getWords();
+
+  console.log(randomSPS);
+
+  //default message if user lose the game
+  var myOutputValue = `The computer chose ${randomSPS}. <br> You chose ${input}. <br><br> You lose! Bummer. <br><br> Now you can type "scissors" "paper" or "stone" to play another round!`;
+
+  //user wins - scissors beats paper, paper beats stone, and stone beats scissors.
+  if (
+    (input == "scissors" && randomSPS == "paper") ||
+    (input == "paper" && randomSPS == "stone") ||
+    (input == "stone" && randomSPS == "scissors")
+  ) {
+    myOutputValue = `The computer chose ${randomSPS}. <br> You chose ${input}. <br><br> Congrats, you win! <br><br> Now you can type "scissors" "paper" or "stone" to play another round!`;
+  }
+
+  //////////////Reverse Game////////////
```

Comment:
we generally want to remove unused/ unwanted code. Commented out code typically fits the need so best to remove commented out code in the future just to keep your code clean and neat